src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -502,6 +502,35 @@ The `type` and default key is `"culture"`.
 
 : _Optional_. The timezone of the locale. For example, `Europe/Vienna`.
 
+## Screenshot Context
+
+The Screenshots Context carries information about screenshots that have been included with an
+event as attachments. 
+
+The `type` and default key is `"screenshot"`.
+
+Each key of `screenshot` should match the filename of an attachment. An SDK can
+include any additional information to helps determine the state of an windows
+and which screenshot is most relevant to the event.
```

Comment:
Can we introduce a more opinionated schema for now?

```ts
type ScreenshotContext = {
  [key: string]: {
    focus?: boolean;
    errored?: boolean;
  } & Record<string, unknown>,
};
```

Reply:
How about this info going in the attachment envelope header?
Somehow being made available to the attachment, the data is used by the thumbnail component in the UI